optimizely/config_manager.py

```diff
@@ -368,3 +370,46 @@ def start(self):
         """ Start the config manager and the thread to periodically fetch datafile. """
         if not self.is_running:
             self._polling_thread.start()
+
+
+class AuthDatafilePollingConfigManager(PollingConfigManager):
```

Comment:
Thinking more about this name. Should we just change this to `AuthenticatedDatafileConfigManager`? Polling is implied by it extending the `PollingConfigManager`.

Reply:
AuthDatafilePollingConfigManager is fine, and it explicitly mentions that this is a polling manager and not static manager.